app/assets/javascripts/administrate/components/associative.js

```diff
@@ -1,3 +1,3 @@
 $(function() {
-  $('.field-unit select').selectize({});
+  $('.field-unit select').not(".non-selectizable select").selectize({});
```

Comment:
I think the following is equivalent since we're already targeting `select` elements?

```suggestion
  $('.field-unit select').not(".non-selectizable").selectize({});
```

Reply:
think, it is not equivalent but it may be preferable in our case.

my thought was to give `non-selectizable` class name to a wrapper div/container, but there might be cases where we will not need the wrapper. so giving `non-selectizable` class name to a `select` should work better for us